Added docs for pre-signed URLs (#783)
e2b-dev/E2B: apps/web/src/app/(docs)/docs/filesystem/upload/page.mdx

```
</CodeGroup>

## Upload with pre-signed URL

Sometimes you may want to upload files to the sandbox from a different environment, that is not authorized, like a user's browser.

> In this case, you can use pre-signed URLs to upload files securely.

Reviewer: For this use case, you can use pre-signed URLs to let users upload files securely.

<CodeGroup>
```js
import fs from 'fs'
import { Sandbox } from '@e2b/code-interpreter'

> // Start secured sandbox (all operations must be authorized by default)

Reviewer 2: ```suggestion
// Start a secured sandbox (all operations must be authorized by default)
```

> // Start secured sandbox (all operations must be authorized by default)

Reviewer: ```suggestion
// Start a secured sandbox (all operations must be authorized by default)
```

const sandbox = await Sandbox.create(template, { secure: true })

// Create a pre-signed URL for file upload with 10 seconds expiration
const publicUploadUrl = sandbox.uploadUrl(
  'demo.txt', {
    useSignature: true,

>     useSignatureExpiration: 10_000, // optionals

Reviewer 2: ```suggestion
    useSignatureExpiration: 10_000, // optional
```

  },
)

> // Upload file with pre-signed URL (this can be executed from any environment such as clients browser)

Reviewer 2: ```suggestion
// Upload file with pre-signed URL (this can be executed from any environment such as a user's browser)
```

const form = new FormData()

> form.append('file', 'file content')

Reviewer 2: ```suggestion
form.append('file', 'file content', 'demo.txt')
```

Author: why?

Reviewer 2: to match the filename above

Reviewer: I think that if you are specifying the upload URL you don't have to do this. @sitole Please check so we are 100% sure when publishing this.

await fetch(publicUploadUrl, { method: 'POST', body: form })

> // File is now available in the sandbox and can be read

Reviewer 2: ```suggestion
// The file is now available in the sandbox and can be read
```
